source/content/partials/decoupled-terminus-plugin-backend.md

```diff
@@ -0,0 +1,143 @@
+
+      ```bash{outputLine}
+      Creating a new site...
+      [notice] Deploying CMS...
+      ```
+
+1. Enter your password when prompted, and select whether you want to proceed or not if you see a message, such as:
+
+    ```bash{outputLines}
+    Now let's create your front-end project...
+    [Exec] Running npm init pantheon-decoupled-kit@canary -- --cmsType d10 --cmsEndpoint=https://dev-donnerstag.pantheonsite.io
+    Need to install the following packages:
+    create-pantheon-decoupled-kit@0.9.2-canary.2
+    Ok to proceed? (y)
+    ```
+
+1. Select the framework generator you want to use.
+
+    ```bash{outputLines}
+    Which generator(s) would you like to run? (Press <space> to select, <a> to toggle all, <i> to
+    invert selection, and <enter> to proceed)
+    ◯ next-drupal
+    ◯ next-drupal-umami-addon
+    ◯ tailwindcss-addon
+    ◯ next-drupal-search-api-addon
+    ```
+
+1. Enter a name for your project when prompted.
+
+    ```bash{outputLines}
+    What is the name of your project?
+    ```
+
+1. Enter the location where you want your output to go when prompted. For example:`(/Users/anitapantheon/anita-site/anita-portfolio)`
+
+    ```bash{outputLines}
+    Where should the output go?
+    ```
+
+1. Select if you want to include additional frameworks, such as Tailwind CSS or not if prompted.
+
+    ```bash{outputLines}
+    Would you like to include tailwindcss? (Y/n) selected y
+    ```
+
+1. Select if you want to continue with your selected changes or not when prompted. You will see a list of the changes you selected.
+
+    ```bash{outputLines}
+    Listing changes for /Users/anitapantheon/anitasite/anita-portfolio/.env.development.local:
+
+    - BACKEND_URL=https://dev-anita-portfolio.pantheonsite.io
+    - CLIENT_ID=
+    - CLIENT_SECRET=
+    - PREVIEW_SECRET=
+    - #DEBUG_MODE=true
+    ? About to overwrite /Users/anitapantheon/anitasite/anita-portfolio/.env.development.local with
+    the changes listed above.
+    Would you like to continue? (Use arrow keys)
+    yes
+    skip
+    yes to all
+    abort
+    ```
+
+    You will receive a message similar to the example below if you select `yes to all`.
+
+    ```bash{outputLines}
+    Your project was generated with:
+    next-drupal
+    cd into /Users/anitapantheon/anitasite/anita-portfolio to start developing!
+    (node:53117) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
+    Goodbye.
+    [Exec] Done in 07:44
+    [notice] Next steps: import your repository to create a Front-End Site https://docs.pantheon.io/guides/decoupled/no-starter-kit/import-repo
+    Your Decoupled Kit project has been created!
+    ```
+
+    - You can see this project in your Pantheon account. Go to your Pantheon dashboard and select **Sites**. You will see your project listed with the **label** name you entered in the steps above.
```

Comment:
I think we need to be more specific that you'll only see the back-end project at this point.